SimpleCalculator2.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+"""
+https://stackoverflow.com/questions/60510955/how-to-store-x-number-of-factors-from-user-input-gtk-textview-in-multi-factor
+"""
+
+import gi
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
+
+import sys
+import operator
+
+
+class Handlers:
+    def __init__(self):
+        print('__init__:()'.format())
+        self.expressionField = builder.get_object("expressionField")
+        self.buffer = self.expressionField.get_buffer()
+        self.buffer.insert(self.end_iter, '5x5 + 2 / 3÷2+ 10 % - 1')
+
+    @property
+    def end_iter(self):
+        return self.buffer.get_end_iter()
+
+    @property
+    def start_iter(self):
+        return self.buffer.get_start_iter()
+
+    def clear(self):
+        self.buffer.delete(self.start_iter, self.end_iter)
+
+    def on_clicked(self, button, data=None):
+        label = button.get_label()
+        print('clicked:{}'.format(label))
```

Comment:
Doesn't this just print out on the console the button that you have clicked?

Reply:
Yes, it's for tracing only.
You can comment it with `# `